silo/management/commands/stripMongoData.py

```diff
@@ -0,0 +1,112 @@
+from django.core.management.base import BaseCommand, CommandError
+from django.utils.encoding import smart_text, smart_str
+from django.conf import settings
+import pymongo
+from bson.objectid import ObjectId
+import json
+import random
+from silo.models import LabelValueStore, Silo
+
+
+class Command(BaseCommand):
+    """
+    Usage: python manage.py update_to_0-9-2
+    """
+    def add_arguments(self, parser):
+        parser.add_argument("--write", action='store_true', dest='write')
+
+    def handle(self, *args, **options):
+
+        client  = pymongo.MongoClient(settings.MONGODB_URI)
+        db = client.get_database(settings.TOLATABLES_MONGODB_NAME)
+
+        counter = 0
+        foundSilos = set()
+        recordsUpdated = []
+        multiMod = []
+
+        # sampleRecords = db.label_value_store.find({"silo_id": {"$in": [175, 891, 996, 981, 1342, 1576]}})
+        # print sampleRecords.count()
+        # for r in sampleRecords:
+        #     print 's=', r['silo_id']
+        # import sys
+        # print 'done'
+        # sys.exit()
+
+        autoPushEnabled = list(Silo.objects.filter(reads__autopush_frequency__isnull=False).values_list('pk', flat=True))
```

Comment:
It's not a blocker, but instead of camel case, in Python PEP8 the style is `auto_push_enabled`